auslib/db.py

```diff
@@ -1847,17 +1844,25 @@ def getReleaseInfo(self, names=None, product=None, limit=None, transaction=None,
                 ((self.db.releases.name == self.db.rules.mapping) | (self.db.releases.name == self.db.rules.fallbackMapping)),
             )
             if transaction:
-                ref_list = transaction.execute(select([self.db.releases.name, self.db.rules.rule_id]).select_from(j)).fetchall()
+                ref_list = transaction.execute(
+                    select([self.db.releases.name, self.db.rules.rule_id, self.db.rules.product, self.db.rules.channel]).select_from(j)
+                ).fetchall()
             else:
-                ref_list = self.getEngine().execute(select([self.db.releases.name, self.db.rules.rule_id]).select_from(j)).fetchall()
+                ref_list = (
+                    self.getEngine()
+                    .execute(select([self.db.releases.name, self.db.rules.rule_id, self.db.rules.product, self.db.rules.channel]).select_from(j))
+                    .fetchall()
+                )
 
             for row in rows:
                 refs = [ref for ref in ref_list if ref[0] == row["name"]]
                 ref_list = [ref for ref in ref_list if ref[0] != row["name"]]
                 if len(refs) > 0:
                     row["rule_ids"] = [ref[1] for ref in refs]
+                    row["rule_info"] = {str(ref[1]): {"product": ref[2], "channel": ref[3]} for ref in refs}
                 else:
                     row["rule_ids"] = []
+                    row["rule_info"] = {}
```

Comment:
Nit: Lines 

```py
row["rule_ids"] = [ref[1] for ref in refs]
row["rule_info"] = {str(ref[1]): {"product": ref[2], "channel": ref[3]} for ref in refs}
```

already handle the case where there's no refs. The `if` statement is not needed 😃